lib/floe/workflow/choice_rule.rb

```diff
@@ -4,24 +4,28 @@ module Floe
   class Workflow
     class ChoiceRule
       class << self
-        def true?(payload, context, input)
-          build(payload).true?(context, input)
-        end
```

Comment:
:+1: good catch looks like this was only used in specs

Reply:
I think this was actually how everything was run.
all and/or/not ended up going this route and creating a rule for each stage.